ember-welcome-page/tsconfig.json

```diff
@@ -1,4 +1,10 @@
 {
   "extends": "@tsconfig/ember/tsconfig.json",
-  "include": ["src/**/*"]
+  "include": [
+    "src/**/*",
+    "unpublished-development-types/**/*"
+  ],
```

Comment:
Copy-pasted from the TypeScript project that `embroider-build/addon-blueprint` generates.